R2API/EliteAPI.cs

```diff
@@ -116,6 +116,10 @@ public static class EliteAPI {
         /// <param name="elite">The elite to add.</param>
         /// <returns>true if added, false otherwise</returns>
         public static bool Add(CustomElite? elite) {
```

Comment:
Now that these methods are so short, they should really get `[MethodImpl(MethodImplOptions.NoInlining)]` put on them for safeties sake, GetCallingAssembly can behave weirdly when methods are inlined

Reply:
Sure thing, i'm a bit curious, what's this stuff of MethodImpl

Reply:
If you check the remarks section in the msdn doc: https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getcallingassembly?view=net-6.0